src/app/core/shared/item-request.model.ts

```diff
@@ -0,0 +1,90 @@
+import { autoserialize, deserialize } from 'cerialize';
+import { typedObject } from '../cache/builders/build-decorators';
+import { excludeFromEquals } from '../utilities/equals.decorators';
+import { ResourceType } from './resource-type';
+import { ITEM_REQUEST } from './item-request.resource-type';
+import { CacheableObject } from '../cache/object-cache.reducer';
+import { HALLink } from './hal-link.model';
+
+/**
+ * Model class for a Configuration Property
+ */
+@typedObject
+export class ItemRequest implements CacheableObject {
```

Comment:
Typedocs are incorrect here :) This is obviously for a model for an Item Request (i.e. Request a copy)